react/wbooks/src/app/screens/Home/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { t } from 'i18next';
+
+import logoWolox from '../../assets/LogoWolox.png';
+
+import styles from './styles.module.scss';
+
+const clearTokens = () => {
+  localStorage.setItem('token_id', '');
+};
```

Comment:
`const clearTokens = () => localStorage.setItem('token_id', '');`

Reply:
I think there is a `localStorage.delete` or something like that 🤔

Reply:
localStorage.clear I think